### PR TITLE
LPS-82021 Security Properties Rename

### PR DESCRIPTION
**Pull request by martamedio**
More info (section Security): https://docs.google.com/document/d/1dKaGiMZF2vYpQ706029RAhkgLfhfG5GNqzsMQBWvPeY/edit?usp=sharing_eip&ts=5b17b57c

Last property OAuth2 Authorize Screen Configuration is no longer present (already changed)

### CODE REVIEW

File: modules/apps/document-library/document-library-document-conversion/src/main/resources/content/Language_ca.properties [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 enabled=Habilitada
 hosts-allowed=Amfitrions permesos
-image-request-auth-verifier-configuration-name=Verificador d'autenticació de sol·licitud d'imatge
 openoffice-configuration-name=Integració d'OpenOffice
-openoffice-server-enabled-help=Habilitar la integració d'OpenOffice permet que el portlet de biblioteca de documents i el portlet Wiki proporcionin la funcionalitat de conversió. Això s'analitza amb OpenOffice 2.3.x a 3.2. x. És recomanable que tingueu l'OpenOffice a la mateixa màquina. Utilitzar un amfitrió remot no és totalment compatible i podria provocar diversos problemes. Per iniciar l'OpenOffice com a servei, executeu l'ordre: soffice -headless -accept="socket,host=127.0.0.1,port=8100;urp;"
+openoffice-server-enabled-help=Enabling OpenOffice integration allows Documents & Media and Wiki to provide conversion functionality. This is tested with OpenOffice 2.3.x through 3.2.x. It is recommended that you have OpenOffice on the same machine. Using a remote host for the instance is not fully supported and could lead to various problems. To start OpenOffice as service, run the command: soffice -headless -accept="socket,host=127.0.0.1,port=8100;urp;" (Automatic Copy)
```
Comment (JorgeFerrer) [inline modules/apps/document-library/document-library-document-conversion/src/main/resources/content/Language_ca.properties]: Why was this change in this pull?

File: modules/apps/document-library/document-library-document-conversion/src/main/resources/content/Language_de.properties [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
 enabled=Aktiviert
 hosts-allowed=Hosts erlaubt
-image-request-auth-verifier-configuration-name=Bildanfrage Authentifizierungsüberprüfung
 openoffice-configuration-name=OpenOffice-Integration
-openoffice-server-enabled-help=Durch das Aktivieren der OpenOffice-Integration können die Portlets Dokumentenbibliothek und Wiki Umwandlung ermöglichen. Das wurde getestet mit OpenOffice 2.3.x bis 3.2.x. Es wird empfohlen, OpenOffice auf demselben Gerät zu haben. Die Nutzung eines Remote-Hosts für die Instanz wird nicht vollständig unterstützt und könnte zu verschiedenen Problemen führen. Um OpenOffice als Dienst zu starten, führen Sie folgenden Befehl aus: soffice -headless -accept="socket,host=127.0.0.1,port=8100;urp;"
+openoffice-server-enabled-help=Enabling OpenOffice integration allows Documents & Media and Wiki to provide conversion functionality. This is tested with OpenOffice 2.3.x through 3.2.x. It is recommended that you have OpenOffice on the same machine. Using a remote host for the instance is not fully supported and could lead to various problems. To start OpenOffice as service, run the command: soffice -headless -accept="socket,host=127.0.0.1,port=8100;urp;" (Automatic Copy)
```
Comment (JorgeFerrer) [inline modules/apps/document-library/document-library-document-conversion/src/main/resources/content/Language_de.properties]: same as above

**Comment by JorgeFerrer** [comment]
Hi Marta,
Thanks for adding the ticket to the commits. What I've noticed is that in some languages the key openoffice-server-enabled-help has been changed, loosing the translation. Do you know why?

**Comment by martamedio** [comment]
@JorgeFerrer I just tested it on a clean master branch and `gw buildLang` changes it after deleting any line of the file `Language.properties` (only happens in the module `document-library-document-conversion`)
It's pretty weird, I can't find the reason why it's doing this 🤔

**Comment by JorgeFerrer** [comment]
Could it be that it's not being used and buildLang is noticing?

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by JorgeFerrer** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/59869. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**